.github/workflows/linux.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        java: [11, 17, 20, 21-ea]
+        java: [11, 17, 20, 21]
```

Comment:
Can we drop 11 and 20 and add 22-ea?

See #2455, where I beat you by a minute or two. ;)

Reply:
Also, this should probably be done for Windows and macOS....

Reply:
I'm on mobile, so things don't move as fast...
Java 11 being a current lts,  I'd keep that, no qualms on dropping 20